core/iwasm/interpreter/wasm_runtime.h

```diff
@@ -79,7 +80,7 @@ struct WASMMemoryInstance {
     /* Module type */
     uint32 module_type;
     /* Shared memory flag */
-    bool is_shared;
+    bh_atomic_32_t ref_count; /* 0: non-shared, >0: reference count */
```

Comment:
Coding style, suggest to change to `> 0`?

Reply:
ok.

Reply:
fixed